packages/babel-cli/src/babel/util.ts

```diff
@@ -10,7 +10,7 @@ import type { FileResult, InputOptions } from "@babel/core";
 export function chmod(src: string, dest: string): void {
   try {
     fs.chmodSync(dest, fs.statSync(src).mode);
-  } catch (err) {
+  } catch (_) {
```

Comment:
[Question] Is there a reason not to:

```suggestion
  } catch {
```

...since you're transpiling from TypeScript anyway?

Reply:
We only enabled the typescript plugin to avoid accidental auto-transformations increasing the output size. :)